src/Providers/LighthouseGraphQLPassportServiceProvider.php

```diff
@@ -30,6 +30,8 @@ public function boot()
 
     public function register()
     {
+        $this->app->register(DependencyInjectionProvider::class);
```

Comment:
here we can just add the singleton that is in the other provider and we save a file.